huawei_vrp/docker/launch.py

```diff
@@ -98,13 +139,14 @@ def bootstrap_spin(self):
             self.logger.trace("OUTPUT: %s" % res.decode())
             # reset spins if we saw some output
             self.spins = 0
-
         self.spins += 1
 
         return
 
     def bootstrap_mgmt_interface(self):
-        self.wait_write(cmd="mmi-mode enable", wait=None)
+        #self.wait_write(cmd="mmi-mode enable", wait=None)
```

Comment:
Would this not break the other Huawei VMs' configuration?

As far as I can tell, this disables the verification prompts for certain configuration commands.

Later, the code also undoes this, so I'm not sure where this would be set.